### PR TITLE
v85.2.0

### PR DESCRIPTION
**Pull request by andrewoverton**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,160 @@
...
+## New features
+
+MDCBanner now supports `intrinsicContentSize`.
+
+```objc
+MDCBannerView *myBanner = ...
+CGSize validIntrinsicContentSize = myBanner.intrinsicContentSize;
+```
+
+MDCTabBarView has several new core features implemented.
```
Comment (jverkoey) [inline CHANGELOG.md:14]: What does core feature mean here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,160 @@
...
+
+```objc
+MDCTabBarView *myTabBarView = ...
+myTabBarView.delegate = self;
+[myTabBarView setTitleFont:myFont forState:UIControlStateNormal];
+[myTabBarView setTitleColor:myColor forState:UIControlStateNormal];
```
Comment (jverkoey) [inline CHANGELOG.md:20]: I can't tell what this code snippet is demonstrating; are these all new APIs?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,160 @@
+# 85.2.0
+
+In this release we have added new features to Banner and Tabs, fixed bugs in Buttons and FlexibleHeader, and improved test coverage in BottomSheet. Additionally, we have added a TextField extension that will eventually house an MDCTextField alternative.
```
Comment (jverkoey) [inline CHANGELOG.md:3]: "In this minor release"

Comment (jverkoey) [inline CHANGELOG.md:3]: BottomNavigation also has a theming extension now.

**Review by jverkoey — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (andrewoverton) [reply CHANGELOG.md:20]: This is a sampling of some of the many new APIs on the new Tabs. I should make that more clear!

File: CHANGELOG.md (hunk 3 above)
Comment (andrewoverton) [reply CHANGELOG.md:3]: Ah thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (andrewoverton) [reply CHANGELOG.md:14]: Things that you'd reasonably expect in any v1 thing but that weren't there before

Comment (andrewoverton) [reply CHANGELOG.md:14]: @romoore suggests just not mentioning it, is that alright? 🤫

Maybe same with MDCBaseTextField?

File: CHANGELOG.md (hunk 2 above)
Comment (andrewoverton) [reply CHANGELOG.md:20]: I got rid of the snippet.

File: CHANGELOG.md (hunk 1 above)
Comment (andrewoverton) [reply CHANGELOG.md:14]: Took out any mention of MDCTabBarView and MDCBaseTextField

**Pull request merged**